src/zalopay/zaloPayClient.ts

```diff
@@ -1,20 +1,17 @@
 import axios, { AxiosInstance } from "axios";
-import { Config } from "./model/Config";
-import { OrderProvider } from "./orderProvider";
+import { Config } from "./utils/Config";
 import { SecurityProvider } from "./securityProvider";
 
 export class ZaloPayClient {
   config: Config;
   httpClient: AxiosInstance;
-  orderProvider: OrderProvider;
   securityProvider: SecurityProvider;
 
   constructor(config: Config) {
     this.config = config;
     this.httpClient = axios.create({
       baseURL: config.env == "sandbox" ? "https://sb-openapi.zalopay.vn" : "https://openapi.zalopay.vn",
     });
-    this.orderProvider = new OrderProvider(this);
     this.securityProvider = new SecurityProvider(this);
```

Comment:
I think `securityProvider` is no longer needed, can we remove?

Reply:
It's used to create a mac string to verify the callback data. I will refactor it later